### DISCUSSION TITLE
Here is my first website on Deta

### DISCUSSION THREAD

**Discussion by Sokhavuth** [deta/cloud-docs#476] [Show and tell]
I would like to share my first experience in building my first website on Deta. This website was built using Node.js 14, Express.js, and Base as the main database. It runs really smooth and fast comparing to other platforms. And I would like to hear from you.
[https://khmerweb-classic-newspaper.deta.dev/](https://khmerweb-classic-newspaper.deta.dev/)
![download](https://user-images.githubusercontent.com/49101948/190028998-3161c12c-d5a3-45d0-931b-6501c959d619.jpeg)

**Comment by abdelhai** [comment]
Looks great! congrats on publishing your first deta website 🙌
